src/components/Popup/Popup.tsx

```diff
@@ -92,60 +105,108 @@ export default class Popup extends UIComponent<Extendable<IPopupProps>, IPopupSt
     'variables',
   ]
 
-  public static defaultProps = {
+  public static defaultProps: IPopupProps = {
     as: Portal,
     align: 'start',
     position: 'above',
+    accessibility: PopupBehavior,
   }
 
-  public state = { triggerRef: undefined }
+  public state = { triggerRef: undefined, popupOpened: false }
+
+  protected actionHandlers: AccessibilityActionHandlers = {
+    performClick: e => this.onTriggerClick(e),
+    closeAndFocusTrigger: e => this.handlePopupState(e, () => false, this.focusTrigger),
+  }
 
   public renderComponent({
     ElementType,
     classes,
+    accessibility,
     rest,
     rtl,
   }: IRenderResultConfig<IPopupProps>): React.ReactNode {
-    const { children, trigger } = this.props
+    const { children, trigger, position, align } = this.props
 
     return (
       <ElementType
         className={classes.root}
         {...rest}
+        open={this.state.popupOpened}
         trigger={trigger}
         triggerRef={this.handleTriggerRef}
+        triggerAccessibility={{
+          attributes: accessibility.attributes.trigger,
+          keyHandlers: accessibility.keyHandlers.trigger,
+        }}
```

Comment:
lets, please, make explicit `as` cast here (and use `TriggerAccessibility` type) - this will ensure that it would be hard to accidentally break the app with new props being added to either attributes or handlers